<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>EuroMilhoes</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>euromilhoes-logo.png</normaloff>euromilhoes-logo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>GET RICH</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>390</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>301</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
